src/screens/tabs/Profile/screens/Contacts/components/AddEditContact/index.js

```diff
@@ -11,10 +11,15 @@ import TextInput from '@/commonComponents/TextInput';
 import UserIcon from '@/commonComponents/UserIcon';
 import RainbowButton from '@/components/buttons/RainbowButton';
 import Text from '@/components/common/Text';
+import ErrorIcon from '@/components/icons/svg/ErrorIcon.svg';
 import { FontStyles } from '@/constants/theme';
 import useICNS from '@/hooks/useICNS';
 import { addContact, editContact } from '@/redux/slices/user';
-import { validateAccountId, validatePrincipalId } from '@/utils/ids';
+import {
+  isICNSName,
```

Comment:
we could rename it to `validateICNSName` to maintain naming in those functions

Reply:
Agree